python/nano/src/bigdl/nano/deps/neural_compressor/pytorch/quantized_model.py

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import os
+
+import yaml
+from bigdl.nano.utils.inference.pytorch.model import AcceleratedLightningModule
+from neural_compressor.utils.pytorch import load
+from neural_compressor.model.model import PyTorchModel
+
+
+class QuantizedModel(AcceleratedLightningModule):
+    def __init__(self, model):
+        super().__init__(model.model)
+        self.quantized = model
+
+    def save(self, path):
+        return self.quantized.save(path)
+
+    def load_state_dict(self, state_dict):
+        load(state_dict, self.quantized._model)
```

Comment:
`quantized._model` or `quantized.model`?

Reply:
1. same thing, copied from inc src code, will use `quantized.model`
2. will add a test to confirm that.

Reply:
Fix this in new commits.